app/views/chitter/index.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Welcome to Chitter! </h1>
+
+<body>
+
+"If you're already a user, please sign in below"
+
+<form class="login" action="/login" method="post">
+  Username:<br>
+  <input type="text" name="username" value=""><br>
+  Password:<br>
+  <input type="password" name="password" value=""><br><br>
```

Comment:
Probably was time issues but usually better to use CSS aligning than mulitple <br>s (I think)